pkg/virt-controller/services/template.go

```diff
@@ -62,6 +70,79 @@ func (t *templateService) RenderLaunchManifest(vm *v1.VM) (*kubev1.Pod, error) {
 	return &pod, nil
 }
 
+func (t *templateService) RenderMigrationJob(vm *v1.VM, sourceNode *kubev1.Node, targetNode *kubev1.Node) (*batch.Job, error) {
+	srcAddr := ""
+	dstAddr := ""
+	for _, addr := range sourceNode.Status.Addresses {
```

Comment:
There's a fundamental difference between the two choices that should be discussed.

If we consider both hostName and interalIP within the same loop, then that means the first address in the Addresses list that is either a hostName or an internalIP will be used.

If the address types are considered in separate loops, then this means that hostName will be used regardless of where is appears in the Addresses list. If and only if hostName is not present at all will internalIP be considered as a candidate for srcAddr.

Which approach would be preferable? @rmohr @fabiand @admiyo